lib/msal-core/src/Storage.ts

```diff
@@ -175,4 +175,20 @@ export class Storage {// Singleton
         this.setItemCookie(Constants.loginRequest, "", -1);
         this.setItemCookie(Constants.stateAcquireToken, "", -1);
     }
+
+    /**
+     * Create acquireTokenUserKey to cache user object
+     * acquireTokenUserKey = Constants.acquireTokenUser + Constants.resourceDelimeter + userId + Constants.resourceDelimeter + authenticationRequest.state;
+     */
+    static generateATUserKey(userId: any, state: string) {
+        return `msal.acquireTokenUser|${userId}|${state}`;
```

Comment:
any reason we are not using the constants here?

Reply:
Nothing particular. I thought this style is simpler, picked it from some String Interpolation examples. We are moving constants from the central location piece by piece so I added text. I can use constants too in the same utils function if we want to adhere to that standard.

Reply:
if the const is specific to a file, then it might make sense to move locally. However if it's used in different files, centralized location is better, correct?

Reply:
technically  this is not a const, it is just a string being generated at run time.

Reply:
@sameerag  can you add return type to the fn

Reply:
Done

Reply:
So I have created a new constant in Constants.ts file for CacheKeys and used them. There is more clean up needed for Constants but we will take that up after the current activity. I have added an internal tracker for this.